testing/tests/DevExpress.ui.widgets/listParts/editingTests.js

```diff
@@ -125,6 +125,33 @@ QUnit.test('index should be correct for grouped list', function(assert) {
     assert.equal(list.getItemElementByFlatIndex(5).get(0), $item.get(0), 'item correct');
 });
 
+QUnit.test('grouped list can select items with index more then 255 (T996851)', function(assert) {
+    const items = [];
+    for(let i = 0; i < 300; i += 1) {
+        items.push(i);
+    }
+    const dataSource = [{
+        key: 'someKey',
+        items
+    }];
+    let selectedIdx;
```

Comment:
let's do not use not generally accepted abbreviated words
```suggestion
    let selectedIndex;
```

Reply:
done